core/src/test/scala/org/apache/spark/scheduler/SchedulerIntegrationSuite.scala

```diff
@@ -648,4 +661,70 @@ class BasicSchedulerIntegrationSuite extends SchedulerIntegrationSuite[SingleCor
     }
     assertDataStructuresEmpty(noFailure = false)
   }
+
+  testScheduler("[SPARK-19263] DAGScheduler shouldn't resubmit active taskSet.") {
```

Comment:
same here on the test name, perhaps "DAGScheduler should not submit multiple active tasksets, even with late completions from earlier stage attempts"